cmd/lncli/commands.go

```diff
@@ -659,6 +659,11 @@ var openChannelCommand = cli.Command{
 				"transaction must satisfy",
 			Value: 1,
 		},
+		cli.Int64Flag{
+			Name: "max_value_in_flight_msat",
```

Comment:
should we prefix with `remote_` to make it clear? not always clear to people that you tell the other party what values to use